docs/zksync-era-integration.md

```diff
@@ -0,0 +1,184 @@
+# ZK stack
+
+## Introduction
+
+So far we have been talking only about the `era_vm`. But you should know that the vm is only a small part of the zk stack. The zk stack is composed of many critical components. In this section, we are only going to be interested in one particular component: the **nodes**, which can further be decomposed into the following units:
```

Comment:
> we are only going to be interested in one particular component: the **nodes**, which can further be decomposed into the following units:

The word *node* is a bit overloaded, I would rephrase this as saying the component we are interested in is the **Operator/Sequencer**, explain what it is and talk about both the bootloader and the VM that it uses